### PR TITLE
Changes to create database resources in contacts namespace

### PR DESCRIPTION
**Pull request by kumaya**

### CODE REVIEW

File: deploy/kube.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ spec:
       - name: server
         args:
         - "-db"
-        - "host=postgres port=5432 user=postgres password=postgres sslmode=disable dbname=contacts"
+        - "host=contactsappdbserver port=5432 user=atlasdbadmin password=Inf@blox sslmode=disable dbname=contactsappdb"
```
Comment (chinmayb) [inline deploy/kube.yaml]: Use `dsn` secrets

File: deploy/contacts-localdb.yaml [hunk 2]
```diff
@@ -28,7 +30,8 @@ apiVersion: atlasdb.infoblox.com/v1alpha1
 kind: DatabaseSchema
 metadata:
   name: contactsappschema
+  namespace: contacts
 spec:
   database: contactsappdb
-  git: github://<USERNAME:<TOKEN>@infobloxopen/atlas-contacts-app/db/migrations
+  git: github://$GIT_USER:$GIT_TOKEN@infobloxopen/atlas-contacts-app/db/migrations
```
[the remark this answers is not in the document]
Comment (kumaya) [reply deploy/contacts-localdb.yaml]: why would we need this ?

[the remark this answers is not in the document]
Comment (chinmayb) [reply deploy/contacts-localdb.yaml]: we need github url there too.

[the remark this answers is not in the document]
Comment (kumaya) [reply deploy/contacts-localdb.yaml]: but that's a secret https://github.com/infobloxopen/atlas-contacts-app/blob/master/deploy/contacts-rds.yaml

File: README.md [hunk 3]
```diff
@@ -10,21 +10,25 @@ These instructions will get you a copy of the project up and running on your loc
...
+    export GIT_TOKEN=<YOUR_GIT_USER_TOKEN>
+    cat deploy/contacts-localdb.yaml | envsubst '$GIT_USER$GIT_TOKEN' | kubectl apply -f -
   ```
+  To create GIT_TOKEN if not exists follow [link](https://github.com/settings/tokens)
+
   * To use RDS database instance, database and schema/tables resources used by contacts app, modify `contacts-rds.yaml` and run"
-  ```sh
+  ```
     kubectl create -f ./deploy/contacts-rds.yaml
   ```
```
[the remark this answers is not in the document]
Comment (jkraj) [reply README.md]: When we reduce the atlas-db-controller sync time this may not need in dev environment

[the remark this answers is not in the document]
Comment (chinmayb) [reply README.md]: okay, understood.

**Pull request merged**